lazyflow/request/request.py

```diff
@@ -1522,3 +1535,54 @@ def clean(self):
         Release our handles to all requests in the pool, for cleanup purposes.
         """
         self._requests = set()
+
+
+class _ValueRequest:
+    """
+    Pseudo request that behaves like a request.Request object.
+
+    This object is used to prevent the heavy construction of complete
+    Request objects in simple cases where they are not needed.
+    """
+
+    def __init__(self, value):
+        self.result = value
+        self.started = False
+
+    def wait(self):
+        return self.result
+
+    def block(self):
+        pass
+
+    def submit(self):
+        pass
+
+    def notify_finished(self, callback):
+        callback(self.result)
+
+    def notify_failed(self, callback):
+        pass
+
+    def notify_cancelled(self, callback):
+        pass
+
+    def clean(self):
+        self.result = None
+
+    def writeInto(self, destination):
+        if isinstance(destination, ma.masked_array):
+            destination.data[...] = ma.getdata(self.result)
+            destination.mask[...] = ma.getmaskarray(self.result)
+            if isinstance(self.result, ma.masked_array):
+                destination.fill_value = self.result.fill_value
```

Comment:
have you tested this code? From the docs i take that `.fill_value` returns the fill value and `.set_fill_value()` is used to set it:
 * https://docs.scipy.org/doc/numpy-1.14.1/reference/maskedarray.baseclass.html#numpy.ma.MaskedArray.fill_value

we currently use numpy `1.14`
`fill_value` as a property is only available as of numpy `1.17.2` (`set_fill_value()` is kept there for compatibility)

Reply:
> we currently use numpy 1.14
fill_value as a property is only available as of numpy 1.17.2 (set_fill_value() is kept there for compatibility)

I assume it is there just wasn't documented as it worked on even older versions of numpy. Added test.

Reply:
yeah sorry, I read the code wrong. But really great that it's tested now :)